query-node/mappings/bootstrap/workers.ts

```diff
@@ -0,0 +1,39 @@
+import { DatabaseManager } from '@dzlzv/hydra-db-utils'
+import { Worker, WorkerType } from 'query-node'
+import {logger} from '../src/common'
+
+export interface IBootstrapWorkers {
+  storage: IBootstrapWorker[]
+  gateway: IBootstrapWorker[]
+}
+
+export interface IBootstrapWorker {
+  role_account_id: string
+}
+
+export async function bootWorkers(db: DatabaseManager, workers: IBootstrapWorkers): Promise<void> {
+  await bootWorkersInGroup(db, workers.storage, WorkerType.STORAGE)
+  await bootWorkersInGroup(db, workers.gateway, WorkerType.GATEWAY)
+}
+
+export async function bootWorkersInGroup(db: DatabaseManager, workers: IBootstrapWorker[], workerType: WorkerType): Promise<void> {
+  if (!workers) {
+    return
+  }
+
+  for (const rawWorker of workers) {
+    // create new membership
+    const worker = new Worker({
+      // main data
+      workerId: rawWorker.role_account_id,
```

Comment:
This should be the `rawWorker.id` field (WorkerId) NOT the role account (AccounId).
See comment on the export script https://github.com/Joystream/runtime-state-export-tools/pull/6#discussion_r628383185

Reply:
Done